db/seeds.rb

```diff
@@ -0,0 +1,19 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+User.destroy_all
+Timeline.destroy_all
+Post.destroy_all
+Profile.destroy_all
+
+
+a = User.new
+a.username = "admin"
+a.email = "admin@admin.com"
+a.password = "d049842392"
+a.password_confirmation = "d049842392"
+a.save
```

Comment:
The admin password is on github, no sure if you care but it looks like you didn't want everyone to see it